lib/excavate/archive.rb

```diff
@@ -240,6 +240,8 @@ def archive?(file)
       return false unless File.file?(file)
 
       ext = normalized_extension(file)
+      return false if ext == "gz" && FileMagic.detect(file) != :gzip
```

Comment:
Instead of this, should we have a method that detects the actual file type using the file's magic number?

Reply:
Oh we do! Closing.